db/changes.go

```diff
@@ -820,3 +820,106 @@ func (db *Database) getLateFeed(feedHandler *lateSequenceFeed) (<-chan *ChangeEn
 func (db *Database) closeLateFeed(feedHandler *lateSequenceFeed) {
 	db.changeCache.getChannelCache(feedHandler.channelName).ReleaseLateSequenceClient(feedHandler.lastSequence)
 }
+
+/*
```

Comment:
Looks like this comment style doesn't conform to the [godoc](https://blog.golang.org/godoc-documenting-go-code) style.

Reply:
Cleaned that up.